src/environment.ts

```diff
@@ -83,7 +53,7 @@ export class SecureEnvironment implements MembraneBroker {
         if (isUndefined(options)) {
             throw ErrorCreate(`Missing SecureEnvironmentOptions options bag.`);
         }
-        const { blueGlobalThis, redGlobalThis, distortionMap } = options;
```

Comment:
this is probably something we can completely eliminate from the env constructor API, since it doesn't use blueGlobalThis anymore, it only needs distortions and red global.